src/js/helpers/nodesHelper.spec.js

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { knotxNodes } from './nodesHelper';
+
+beforeEach(() => {
+    document.body.innerHTML = '';
+})
+
+test('Empty DOM results in empty fragment array', () => {
+    const fragments = knotxNodes()
+    expect(fragments).toStrictEqual([])

```

Comment:
I think you can skip making variables like that, just test output of knotxNodes call itself